src/mono/nuget/Microsoft.NET.Runtime.MonoTargets.Sdk/Sdk/Sdk.props

```diff
@@ -1,4 +1,8 @@
 <Project>
+  <PropertyGroup>
```

Comment:
As I understand, coreclr is going to be the default target for xamarin macos.  So, blanket setting this property is not going to work.  

There is the ability to switch runtimes, but I'm not sure if that is going to be a supported feature or not.

Reply:
Hmm so, RID=browser-wasm for now?